tests/com/goide/editor/GoHighlightingAnnotatorTest.java

```diff
@@ -37,23 +37,23 @@ public void testSimple() {
   public void testLabel() {
     doTest();
   }
-  
```

Comment:
changes in this file don't look like `Adding check third index in string slices`